src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Builds.java

```diff
@@ -9,30 +9,6 @@
 import java.util.*;
 
```

Comment:
Thanx for doing that. Jsut fore rcord, new swithc, saying how much buolds to find, will arrive, oook?